src/WebJobs.Script/Description/DotNet/FSharp/FSharpCompilationService.cs

```diff
@@ -196,6 +203,8 @@ public ICompilation GetFunctionCompilation(FunctionMetadata functionMetadata)
             return new FSharpCompilation(errors, assemblyOption);
         }
 
+        static bool _isMono = Type.GetType("Mono.Runtime") != null;
```

Comment:
We should probably move the `PlatformHelper` to `WebJobs.Script` and use it here as well, so we can remove this and have that logic centralized there.